handler/PostMessageSentHandler.ts

```diff
@@ -100,8 +100,13 @@ export class PostMessageSentHandler {
                                 this.modify,
                                 { text: serviceUnavailable ? serviceUnavailable : DefaultMessage.DEFAULT_DialogflowServiceUnavailableMessage });
 
-            updateRoomCustomFields(rid, { isChatBotFunctional: false }, this.read, this.modify);
             const targetDepartment: string = await getAppSettingValue(this.read, AppSetting.FallbackTargetDepartment);
+            if (!targetDepartment) {
+                console.error('Failed to handover: Handover Department not configured');
```

Comment:
Add this Log in `enum/Logs.ts` file and reuse it at every place

Reply:
Thanks @Shailesh351, Updated the PR